### PR TITLE
chore: fix CI

### PR DESCRIPTION
**Pull request by czarcas7ic**
## Description

Fixes some lingering issues that have been preventing the CI from properly working.

### CODE REVIEW

File: tests/integration/bank/keeper/deterministic_test.go [hunk 1]
```diff
@@ -218,7 +218,7 @@ func (suite *DeterministicTestSuite) TestGRPCQueryTotalSupply() {
 	suite.Require().NoError(suite.bankKeeper.MintCoins(suite.ctx, minttypes.ModuleName, coins))
 
 	req := &banktypes.QueryTotalSupplyRequest{}
-	testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.TotalSupply, 243, false)
+	testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.TotalSupply, 3285, false)
```
Comment (czarcas7ic) [inline tests/integration/bank/keeper/deterministic_test.go:221]: This was strange to me, although I know we modified the bank query logic so it makes sense the gas used is different.

File: tests/integration/bank/keeper/deterministic_test.go [hunk 2]
```diff
@@ -238,7 +238,7 @@ func (suite *DeterministicTestSuite) TestGRPCQueryTotalSupplyOf() {
 
 	suite.Require().NoError(suite.bankKeeper.MintCoins(suite.ctx, minttypes.ModuleName, sdk.NewCoins(coin)))
 	req := &banktypes.QuerySupplyOfRequest{Denom: coin.GetDenom()}
-	testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.SupplyOf, 1021, false)
+	testdata.DeterministicIterations(suite.ctx, suite.Require(), req, suite.queryClient.SupplyOf, 2033, false)
```
Comment (czarcas7ic) [inline tests/integration/bank/keeper/deterministic_test.go:241]: Same here

File: tests/e2e/auth/vesting/suite.go [hunk 3]
```diff
@@ -221,6 +223,11 @@ func (s *E2ETestSuite) TestNewMsgCreatePermanentLockedAccountCmd() {
 }
 
 func (s *E2ETestSuite) TestNewMsgCreateClawbackVestingAccountCmd() {
+	pwd, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(pwd)
```
Comment (mattverse) [inline tests/e2e/auth/vesting/suite.go]:
```suggestion
```

**Review by mattverse — APPROVED** [review]
ACK thanks for fixing this 🙇

File: tests/e2e/auth/vesting/suite.go (hunk 3 above)
Comment (czarcas7ic) [reply tests/e2e/auth/vesting/suite.go]: oh we need this, just the print can go away

**Comment by mattverse** [comment]
@czarcas7ic lol sorry for deleting the whole thing, thanks. Planning to merge this as soon as CI passes

**Comment by czarcas7ic** [comment]
Thanks, my hope is the labeler will get fixed after merged, i think its a perms thing that requires it to be in main

**Comment by czarcas7ic** [comment]
Lets remove the sonar cloud step from repo-analysis

**Comment by mattverse** [comment]
@czarcas7ic I'll do that in a subsequent PR

**Comment by czarcas7ic** [comment]
oh lol im sorry @mattverse i just did it

**Comment by mattverse** [comment]
@czarcas7ic GOAT

**Pull request merged**